docs/installation/presto-configuration.rst

```diff
@@ -3,6 +3,10 @@
 Configuring Presto
 ==================
 
+
+Memory configuration
```

Comment:
This header should go after the first paragraph. And then the second paragraph doesn't need "For example"